docs/src/data/hid.js

```diff
@@ -3590,7 +3590,7 @@ export default [
     os: {
       windows: null,
       linux: true,
-      android: false,
+      android: true,
```

Comment:
```suggestion
      android: false,
```
Tested and found that this does not work on my device (Android 14)